### PR TITLE
AAP-4999 Add attribute for RHSSO

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Add an attribute for RHSSO to future-proof the AAP docs in case of product name changes.
AAP-4999

### CODE REVIEW

**Comment by ariordan-redhat** [comment]
Should we have a separate attribute for the RHSSO Operator?  Ideally, we shouldn't need it.

This is just a suggestion, in case a name change for the RHSSO product isn't immediately reflected in the RHSSO Operator name on the OCP UI.

File: downstream/attributes/attributes.adoc [hunk 1]
```diff
@@ -101,3 +101,7 @@
 :OCPLatest: 4.10
 :ODF: Red Hat OpenShift Data Foundation
 :ODFShort: OpenShift Data Foundation
+
+// Red Hat products
+:RHSSO: Red Hat Single Sign-On
+:RHSSOOperator: Red Hat Single Sign-On Operator
```
Comment (oraNod) [inline downstream/attributes/attributes.adoc:107]:
```suggestion
:OperatorRHSSO: Red Hat Single Sign-On Operator
```

Comment (oraNod) [reply downstream/attributes/attributes.adoc:107]: I think this should follow the convention of the other [Operator attributes](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/blob/343e0a70fc5489d6f70faf5f04531985b2961346/downstream/attributes/attributes.adoc#L22)

Comment (ariordan-redhat) [reply downstream/attributes/attributes.adoc]: @oraNod Done. Thanks.

**Review by oraNod — APPROVED** [review]

**Pull request merged**